nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/reporting/StandardReportingInitializationContext.java

```diff
@@ -113,4 +113,5 @@ public String getControllerServiceName(final String serviceIdentifier) {
     public ComponentLog getLogger() {
         return logger;
     }
+
```

Comment:
nit-pick: this is the only change to this file (adding a space at the end). Can this be removed?